website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -2660,17 +2660,28 @@ output_pc = input_pc | beam.Map(lambda item: beam.Row(bank=item["bank"],
 {{< /highlight >}}
 
 {{< paragraph class="language-py" >}}
-Note that this declaration doesn't include any specific information about the
-types of the `bank` and `purchase_amount` fields. Beam will attempt to infer
-type information, if it's unable to it will fall back to the generic type
+Sometimes it can be more concise to express the same logic with the
+[`Select`](https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.core.html#apache_beam.transforms.core.Select) transform:
```

Comment:
Note this link is broken today. Let's hold off on merging until 2.25.0 is out and this link works.